src/opencv.mk

```diff
@@ -52,12 +54,12 @@ define $(PKG)_BUILD
     # openexr isn't available on x86_64-w64-mingw32
     # opencv builds it's own libIlmImf.a
     $(if $(findstring x86_64-w64-mingw32,$(TARGET)),\
-        $(SED) -i 's/OpenEXR//' '$(BUILD_DIR)/unix-install/opencv.pc')
-    $(SED) -i 's,share/OpenCV/3rdparty/,,g' '$(BUILD_DIR)/unix-install/opencv.pc'
-    $(INSTALL) -m755 '$(BUILD_DIR)/unix-install/opencv.pc' '$(PREFIX)/$(TARGET)/lib/pkgconfig'
-
+    $(SED) -i 's/OpenEXR//' '$(1).build/unix-install/opencv.pc')
```

Comment:
Why did you replace `$(BUILD_DIR)` with `$(1).build`? The later is deprecated.

Reply:
Didn't mean to, I've got a merge conflict gone wrong; will probably end-up opening a new PR shortly